build.sbt

```diff
@@ -1,95 +1,43 @@
 
 name := "sPDF"
 
-description := "Create PDFs using plain old HTML+CSS. Uses wkhtmltopdf on the back-end which renders HTML using Webkit."
-
-homepage := Some(url("https://github.com/cloudify/sPDF"))
+version := "1.4.0"
```

Comment:
Wasnt it 1.4.1 already?

Reply:
The latest published version is `1.4.0`. I think once we merged and published it will be `1.4.1`?